pkg/utils/constants.go

```diff
@@ -47,25 +47,25 @@ const (
 	// The key of SnapshotManager location
 	VolumeSnapshotterManagerLocation = "SnapshotManagerLocation"
 	// Valid values for the config with the VolumeSnapshotterManagerLocation key
-	VolumeSnapshotterPlugin = "Plugin"
+	VolumeSnapshotterPlugin     = "Plugin"
```

Comment:
Why extra spaces here?

Reply:
gofmt did that, and matches with line 51.